### ISSUE TITLE
Testing: Linked Data Services

### ISSUE THREAD

**Issue by jimsafley** [omeka-s-modules/UriDereferencer#4]
The best way to test this module is to confirm that the handful of services that come with it are working properly. Add an item, and, for each service below, save several resource URIs as Omeka URI values:

- [DBpedia](http://dbpedia.org/fct/): do a search, click on entity that begins with "dbr:", click on "Goto", use that URL
- [Geonames](https://www.geonames.org): do a search, click on a name, use that URL
- [Getty Vocabularies](http://vocab.getty.edu/): do a search (use AAT, TGN, and ULAN), click on a subject, use "Source" URL
- [LC Linked Data Service](http://id.loc.gov/): click on any of the links, do a search, click on a label, use that URL or a http URL under "URI(s)"
- [OCLC VIAF](http://viaf.org/): do a search, click on a heading, use "Permalink" URL
- [OCLC FAST](http://fast.oclc.org/searchfast/): do a search, click on a heading, click on the "Permanent Link", use that URL
- [RDA Value Vocabularies](http://www.rdaregistry.info/termList/): click on any vocabulary, click on a CURIE, use the "@id" URL
- [Wikidata](https://www.wikidata.org/wiki/Wikidata:Main_Page): do a search, click on any option, use that URL

Then, on a public site, go to the item and you should see "[+]" next to each URI. Click on this and you should see some dereferenced data about the resource.

**Comment by mebrett** [comment]
Working through the services, but a note that the `[+]` does not display in the theme CenterRow, and the styling on the Default theme is a little hard to parse.

**Comment by mebrett** [comment]
All of them worked as expected. I would recommend we check the public side styling to make sure it's accessible (both in terms of screen readers and information legibility)

**Comment by jimsafley** [comment]
@kimisgold See Megan's notes above about styling issues on public themes. We probably need to hold off on releasing this module until those are resolved.

**Comment by kimisgold** [comment]
@jimsafley For some reason, the collapsible content and its toggle aren't appearing in the Center Row theme. I'm having trouble tracking down the cause.

**Comment by kimisgold** [comment]
The public markup needs `aria-controls` and `aria-expanded` attributes on the toggle button, as well as `id`s on the collapsible content (to relate `aria-controls` toggles to their content). ([Reference](https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions#Example_1:_Using_a_button_to_collapse_and_expand_a_region))
